cockroachcloud/monitoring-page.md

```diff
@@ -72,6 +72,22 @@ The [available metrics](#available-metrics) are drawn directly from the Cockroac
 Metric values and time-series graphs in Datadog are not guaranteed to match those in the [DB Console](#access-the-db-console), due to differences in how CockroachDB and Datadog calculate and display metrics.
 {{site.data.alerts.end}}
 
+#### Enable percentiles for selected metrics
+
+A subset of CockroachDB metrics require that you explicitly enable percentiles for them in the Datadog interface, or graphs that display data for them will fail to render properly.
+
+To enable percentiles for these metrics, perform the following steps:
+
+1. In your Datadog interface, select **Metrics** then **Summary** from the sidebar.
```

Comment:
I question the general practice of giving such detailed documentation for operations in another company's product's UI. It does sound like the setting is pretty buried, so I'm sure this level of detail would be helpful while it remains accurate, but Datadog could change those details out from under us, rendering the instructions more confusing than helpful. This is not a hard and fast objection, so feel free to keep it as is if you consider the benefits worth the cost in this particular case.
 
Checking the "distribution" checkbox seems like a useful and non-obvious pro tip, so I would suggest keeping that.